web/cypress.config.ts

```
@@ -0,0 +1,136 @@
+import { defineConfig } from 'cypress';
+import registerCypressGrep from '@cypress/grep/src/plugin';
+const fs = require('fs');
+const path = require('path');
+const report_dir = process.env.ARTIFACT_DIR || '/tmp';
+
+export default defineConfig({
+  screenshotsFolder:  path.join(report_dir, 'cypress', 'screenshots'),
+  screenshotOnRunFailure: true,
+  trashAssetsBeforeRuns: true,
+  videosFolder: path.join(report_dir, 'cypress', 'videos'),
+  video: true,
```

Biome (2.4.4)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)